tests/symbolicator/__init__.py

```diff
@@ -97,15 +97,14 @@ def insta_snapshot_native_stacktrace_data(self, event, **kwargs):
     )
 
 
-def insta_snapshot_javascript_stacktrace_data(self, event, **kwargs):
+def insta_snapshot_javascript_stacktrace_data(insta_snapshot, event):
```

Comment:
Same as above, we don't inherit this method anymore and need to pull it from fixtures.